.github/meta.yaml

```diff
@@ -52,7 +52,7 @@ outputs:
       imports:
         - evalml
       requires:
-        - pytest ==6.2.5
+        - pytest >=6.2.5
```

Comment:
This can be dangerous :) at least make sure it cannot be bigger then 7.0.0, i.e, <7.0.0

Reply:
Thanks for the warning! I'm curious, what exactly makes 7.0.0 dangerous for pytest?

Reply:
Nothing. But is a good things to do. Let's say that tomorrow they piblish a new version of pytest which break everything. If you bound the version above, then you are protect against that 😃 

I think it is always a good idea to bound versions (not to pin this is way too strict)

But just an advice ;) 

Pytest will be for sure not a problem but other packages could give this problem.

Reply:
Sounds good, thanks for the explanation!

Reply:
@MichaelFu512 If you look at the CI logs you can see we are already running with pytest `8.2.1`, so there is definitely no need to restrict this to `<7.0.0`.

Example snippet:
```
Successfully installed attrs-23.2.0 coverage-7.5.3 debugpy-1.8.1 evalml-0.83.0 execnet-2.1.1 fastjsonschema-2.19.1 iniconfig-2.0.0 ipykernel-6.29.4 jsonschema-4.22.0 jsonschema-specifications-2023.12.1 jupyter-client-8.6.2 jupyter-core-5.7.2 nbformat-5.10.4 nbval-0.11.0 nest-asyncio-1.6.0 pluggy-1.5.0 pytest-8.2.1 pytest-cov-5.0.0 pytest-timeout-2.3.1 pytest-xdist-3.6.1 referencing-0.35.1 rpds-py-0.18.1
```